pyfr/integrators/dual/pseudo/multip.py

```diff
@@ -23,6 +25,8 @@ def __init__(self, backend, systemcls, rallocs, mesh, initsoln, cfg,
 
         # Get the multigrid cycle
         self.cycle, self.csteps = zip(*cfg.getliteral(mgsect, 'cycle'))
+        self._fgen = np.random.default_rng(0)
```

Comment:
For reproducibility it might be worth being explicit about the generator

Reply:
For example
```python
seed = self.cfg.getint('solver', 'seed', 1234)
self._fgen = np.random.Generator(np.random.MT19937(seed))
```

Reply:
Freddie said not to expose the seed: https://github.com/PyFR/PyFR/pull/362#discussion_r1296828282